### ISSUE TITLE
Retired courses are missing meta include

### ISSUE THREAD

**Issue by rediris** [gymnasium/tracker#225]
View source on the following pages to see which meta component needs to be replaced.

Examples:
* https://thegymnasium.com/courses/GYM/105/0/
* Error message `<!-- Error loading: https://thegymcms.com/courses/full/gym-105/meta -->`

This type of thing will need to be done for each and every retired course that doesn't already have a meta file.
